specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/dscpConfiguration.json

```diff
@@ -0,0 +1,344 @@
+              "type": "string",
+              "description": "The name of the resource."
+            },
+            {
+              "$ref": "./network.json#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+            }
+          ],
+          "responses": {
+            "204": {
+              "description": "Request successful. Resource does not exist."
+            },
+            "202": {
+              "description": "Accepted and the operation will complete asynchronously."
+            },
+            "200": {
+              "description": "Delete successful."
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "./network.json#/definitions/CloudError"
+              }
+            }
+          },
+          "x-ms-examples": {
+              "Delete DSCP Configuration": {
+                "$ref": "./examples/DscpConfigurationDelete.json"
+              }
+          },
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
+              "final-state-via": "location"
+          }
+        },
+        "get":{
+          "tags": [
+            "DscpConfiguration"
+          ],
+          "operationId": "DscpConfiguration_Get",
+          "description": "Gets a DSCP Configuration",
+          "parameters": [
+            {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group."
+            },
+            {
+              "name": "dscpConfigurationName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource."
+            },
+            {
+              "$ref": "./network.json#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "Request successful. The operation returns the resulting DSCP Configuration resource.",
+              "schema": {
+                "$ref": "#/definitions/DscpConfiguration"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "./network.json#/definitions/CloudError"
+              }
+            }
+          },
+          "x-ms-examples": {
+            "Get Dspc Configuration": {
+              "$ref": "./examples/DscpConfigurationGet.json"
+            }
+          },
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
```

Comment:
Is get method a long running operation as well? It's a little bit rare and just want to confirm.

Reply:
Removing it. Double checked with how the networkInterface.json handles its getters.